consensus/src/chained_bft/consensus_types/timeout_msg.rs

```diff
@@ -126,10 +158,16 @@ impl FromProto for PacemakerTimeout {
         let round = object.get_round();
         let author = Author::try_from(object.take_author())?;
         let signature = Ed25519Signature::try_from(object.get_signature())?;
+        let vote = if let Some(vote_msg) = object.vote.into_option() {
```

Comment:
nit: possible alternative:
`let vote = object.vote.into_option().map(VoteMsg::from_proto)?`

Reply:
I didn't find a way to apply the '?' operator on the Result that is within the Option, hence had to open it up...